hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/BlockOutputStreamEntry.java

```diff
@@ -223,7 +223,7 @@ long getLength() {
    * Gets the block token that is used to authenticate during the write.
    * @return the block token for writing the data
    */
-  Token<OzoneBlockTokenIdentifier> getToken() {
```

Comment:
Not sure it's a good idea to make this public as we return token object? 
I have an idea to avoid make them public. Create a small helper class in the same package in test and add getToken and getCurrentPosition APIs and return the actual. 

class name at test: *.client.io.BlockStreamAccessor

Reply:
Oh, sounds a good idea, I'll try that, thanks~